src/products/iOS_products/review_iOSstrings.md

```diff
@@ -1,14 +1,14 @@
-# Reviewing strings for a new release of Firefox for iOS
+# Reviewing strings for a new release of a Mozilla iOS product
 
-Usually, a new release of Firefox for iOS means an update to strings. If this is the case, a pull request will be made by the iOS team, in order to land these new strings in the [mozilla-l10n firefoxios-l10n repository](https://github.com/mozilla-l10n/firefoxios-l10n).
+Usually, a new release of our iOS products means an update to strings. If this is the case, a pull request will be made by the iOS team, in order to land these new strings, either in the [mozilla-l10n firefoxios-l10n repository](https://github.com/mozilla-l10n/firefoxios-l10n) or the [mozilla-l10n focusios-l10n repository](https://github.com/mozilla-l10n/focusios-l10n).
```

Comment:
I'd shorten the text in the links, it makes them harder to read
https://github.com/mozilla-l10n/documentation/blob/07911271ddc48e268347ddc7c000477e9976fb68/src/products/iOS_products/review_iOSstrings.md

```suggestion
Usually, a new release of our iOS products means an update to strings. If this is the case, a pull request will be made by the iOS team, in order to land these new strings, either in the [firefoxios-l10n](https://github.com/mozilla-l10n/firefoxios-l10n) or the [focusios-l10n](https://github.com/mozilla-l10n/focusios-l10n) repositories in `mozilla-l10n`.
```